graph/testgraph/testgraph.go

```diff
@@ -48,6 +48,27 @@ func checkEmptyIterator(t *testing.T, it graph.Iterator, useEmpty bool) {
 	}
 }
 
+// Edge is supports the simples edge operations.
```

Comment:
Delete "is", and is it "simple" or "simplest"?

Reply:
`// Edge supports basic edge operations.`